tests/driver_at86rf2xx_aes/Makefile

```diff
@@ -13,11 +13,13 @@ ifneq (,$(filter mulle,$(BOARD)))
   DRIVER := at86rf212b
 endif
 
+TEST_ON_CI_WHITELIST += samr21-xpro iotlab-m3 mulle
```

Comment:
Should this only consider boards actually present in the CI? Or should we just whitelist all boards that would be supported, in case more boards get wired up later on? In the latter case, the following boards should also be added:

`fox` has an `at86rf231` according to `Makefile.dep` an `at86rf231`, `hamilton` an `at86rf233`, `samr30-xpro` an `at86rf212b`, `iotlab-m3` and `iotlab-a8-m3` have an `at86rf231`, and `openmote-b` an `at86rf215`.

Reply:
It will compile on any board, but the run-time test will of course fail without a chip.
So maintaining a list of only the boards physically present on CI should be enough.

(at86rf215 dropped the crypto acceleration btw, but it’s handled by a different driver anyway)

Reply:
OK